cmd/thanos/receive.go

```diff
@@ -112,6 +117,12 @@ func registerReceive(app *extkingpin.App) {
 			return errors.New("no external labels configured for receive, uniquely identifying external labels must be configured (ideally with `receive_` prefix); see https://thanos.io/tip/thanos/storage.md#external-labels for details.")
 		}
 
+		// Check if the YAML configuration of request.logging is correct. .
```

Comment:
```suggestion
		// Check if the YAML configuration of request.logging is correct.
```

Reply:
Let's be consistent, we don't have this comment everywhere else we use `decide`, so maybe not needed?

Reply:
Okay, lets remove them